### ISSUE TITLE
Can't <ctrl-c> bpftrace with high frequency updates

### ISSUE THREAD

**Issue by tyroguru** [iovisor/bpftrace#928]
I thought this was fixed but today I simply did this:

```
# bpftrace --v
bpftrace v0.9.1-116-g5271

# cat syscalls.bt
tracepoint:syscalls:sys_enter_write
{
  printf("%s\n", stack());
}
```

and the session is complete un-interruptible from the command line:

```
# bpftrace ./syscalls.bt > h
warning: stack is deprecated and will be removed in the future. Use kstack instead.
^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
```

What's supposed to be the current situation with this?

**Comment by fbs** [comment]
Can you try with #831 ?

**Comment by tyroguru** [comment]
@fbs - thanks. The bpftrace I reported against was not that old but I just pulled from master and tried it again and the results are the same.

**Comment by fbs** [comment]
@tyroguru  did you pull in the changes from that PR (they're not in master yet)? I cannot reproduce the issue with those.

**Comment by tyroguru** [comment]
Thanks @fbs . I just pulled the master which has your fix for #831 and this now works fine. Thanks much for the fix!
